UnitystationLauncher/Views/LoginView.xaml

```diff
@@ -0,0 +1,56 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d" d:DesignWidth="600" d:DesignHeight="400"
+             VerticalAlignment="Center"
+             x:Class="UnitystationLauncher.Views.LoginView">
+    <StackPanel>
+        <Border BorderThickness="1" BorderBrush="#c5c5c5" />
+
+        <!-- Login form content -->
+        <StackPanel>
+            <StackPanel.Background>
+                <SolidColorBrush Color="White" Opacity="0.07" />
+            </StackPanel.Background>
+            <StackPanel Orientation="Horizontal" Height="60" Margin="20,0,60,0" HorizontalAlignment="Center">
+                <Viewbox Margin="5">
+                    <Image Source="/Assets/Ian512.png" />
+                </Viewbox>
+                <TextBlock VerticalAlignment="Center" Margin="5" FontSize="44">Login</TextBlock>
+            </StackPanel>
+            <TextBox Text="{Binding Email}" Margin="15" Watermark="Email" MaxWidth="400">
+                <TextBox.KeyBindings>
+                    <KeyBinding Command="{Binding Login}" Gesture="Return" />
+                </TextBox.KeyBindings>
+            </TextBox>
+            <TextBox Text="{Binding Password}" Margin="15" PasswordChar="*" Watermark="Password" MaxWidth="400">
+                <TextBox.KeyBindings>
+                    <KeyBinding Command="{Binding Login}" Gesture="Return" />
+                </TextBox.KeyBindings>
+            </TextBox>
+
+            <Grid ColumnDefinitions="*, 1*" HorizontalAlignment="Center" Margin="5" MaxWidth="400">
+                <Button Command="{Binding Create}" Grid.Column="1" Padding="5" Margin="15" HorizontalAlignment="Center"
+                        MinHeight="50" MinWidth="130" Classes="SmallButton" FontSize="24">
+                    Create
+                </Button>
+                <Button Command="{Binding Login}" Grid.Column="0" Padding="5" Margin="15" HorizontalAlignment="Center"
+                        MinHeight="50" MinWidth="130" Classes="SmallButton" FontSize="24">
+                    Login
+                </Button>
+            </Grid>
+            <Button Command="{Binding ForgotPw}" Padding="5" Margin="0,5,0,25" HorizontalAlignment="Center"
+                    MinWidth="100">
+                <Button.Template>
+                    <ControlTemplate>
+                        <TextBlock FontSize="22" Foreground="#4f769d">
+                            Forgotten Password?
```

Comment:
Don't you mean "Forgot Password?"

Reply:
I think this is on the login screen, Past tense should be fine, 
Since you've lost the memory of your password